code/datums/star_economy.dm

```diff
@@ -0,0 +1,326 @@
+/datum/star_resource
+	var/name = "fire the coders"
+	var/cname = "resource"
+
+	var/scale_price = 100000
+	var/scale_weight = 500
+
+
+/datum/star_resource/iron
+	name = "iron ore"
+	cname = "iron"
+
+	scale_price = 40000
+	scale_weight = 2000
+
+/datum/star_resource/hyperlite
+	name = "hyperlite crystals"
+	cname = "hyper"
+
+	scale_price = 200000
+	scale_weight = 150
+
+/datum/star_resource/silicon
+	name = "silicate ore"
+	cname = "silicon"
+
+	scale_price = 60000
+	scale_weight = 1000
```

Comment:
thanks vivalas, Idk if you did this cuz I asked for it or you did it cause you wanted to, but this rocks :)